.github/workflows/build.yml

```diff
@@ -34,3 +34,60 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
We should run `v2` here with `persist-credentials: false` as it's the latest version; https://github.com/JamesIves/github-pages-deploy-action/blob/dev/.github/workflows/integration.yml#L42-L45